Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/EventFilter.cs

```diff
@@ -1,16 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Contoso.Forms.Puppet.UWP;
 using Microsoft.AppCenter;
 using Microsoft.AppCenter.Analytics.Ingestion.Models;
 using Microsoft.AppCenter.Channel;
 using System;
 using System.Threading.Tasks;
+using Xamarin.Forms;
 
+[assembly: Dependency(typeof(EventFilter))]
```

Comment:
EventFilter is not supposed to handle crashes, this is not the right place.

Reply:
Thank you for your suggestion, I have removed the clear crash user confirmation from EventFilter file, and add ClearCrashes file to handle crashes in a new commit.